### PR TITLE
Style Update

### PR DESCRIPTION
**Pull request by brandonhaslegs**
Just wanted to change a few styles to make things feel more modern

- Update all border radii to be the same.
- Update all border radii to rems instead of px
- increase roundness of border radius globally

### CODE REVIEW

File: ui/Screen/Project/Source/CheckoutButton.svelte [hunk 1]
```diff
@@ -30,9 +30,9 @@
     margin-top: 3rem;
     right: 0;
     position: absolute;
-    border-radius: 0.5rem;
+    border-radius: 1rem;
     background: var(--color-background);
-    border: 1px solid var(--color-foreground-level-3);
+    /* border: 1px solid var(--color-foreground-level-3); */
```
Comment (rudolfs) [inline ui/Screen/Project/Source/CheckoutButton.svelte:35]: If this is intentional, we should just delete the line instead of commenting it out.

Comment (brandonhaslegs) [reply ui/Screen/Project/Source/CheckoutButton.svelte:35]: Ah not intentional. Will do

**Comment by juliendonck** [comment]
This is so nice 😍 
Screenshot 2021-06-02 at 17 11 40

**Review by juliendonck — APPROVED** [review]
🤙

**Review by rudolfs — APPROVED** [review]
utACK

**Pull request merged**